jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsListViewImpl.java

```diff
@@ -38,6 +42,7 @@
 @Dependent
 @Templated
 public class CaseActionsListViewImpl extends AbstractView<CaseActionsPresenter> implements CaseActionsPresenter.CaseActionsListView {
+    private int PAGE_SIZE = 4;
```

Comment:
Something tricky here, the Available actions column should be taller to match In Progress and Completed columns. One way to fix it could be adding some extra info for each line in column, that could be the stage as @bsremac suggested. I can think of three status here:
- Dynamic (For user task and process task)
- Stage name (For ad-hoc tasks from a specific case stage)
- Case level (For ad-hoc tasks from the case instance level)
wdyt?